ci/requirements-py27.yml

```diff
@@ -8,3 +8,6 @@ dependencies:
   - xarray
   - matplotlib
   - dask
+  - netcdf4
+  - hdf5
+  - hdf4
```

Comment:
@xylar, can you please clarify why both `hdf4` and `hdf5` are needed?  If I understand correctly, only one of these libraries will be used in `netcdf4` and `xarray`.